### PR TITLE
Share records and make sure they won't fill up disk space

### PR DESCRIPTION
**Pull request by godsic**
It is more convinient for end users to have access to FM/IQ records via http interface. So lets move those to `_share/`. Also, check disk space before storing those files and make sure to leave some headroom (currently 10%) to avoid running out-of-disk-space and, potentially, bricking the systems.

### CODE REVIEW

**Comment by godsic** [comment]
@redradist Please comment if you agree on this.

**Comment by godsic** [comment]
@krakenrf This is WIP until further notice.

File: _signal_processing/signal_utils.py [hunk 1]
```diff
@@ -80,3 +83,8 @@ def write_wav(f, sample_freq, x):
     from scipy.io.wavfile import write
 
     write(f, sample_freq, x)
+
+
+def ok_to_store_file(path: str) -> bool:
```
Comment (redradist) [inline _signal_processing/signal_utils.py:88]: I would rename function starting with verbs like `is`, `has`, `can` and etc.
Could you rename it to `can_store_file`

**Comment by godsic** [comment]
@krakenrf Please have a look at the PR

**Comment by krakenrf** [comment]
@godsic something in this code appears to block the spectrum from updating when there is no signal. If you go into the spectrum  and select an intermittent signal you can see the the entire spectrum stops when there is no signal in the VFO. 

This is also the case when FM demod is turned off.

**Comment by godsic** [comment]
@krakenrf I fixed the problem, could you please give it another try?

**Pull request merged**